data/core/editor/help.cfg

```diff
@@ -463,9 +463,9 @@ In scenario mode, the button accesses an editor for individual schedules for <re
 
 There is a filter function to show only a subset of the available items — this is the leftmost of the four buttons at the top of the palette, and the graphic changes depending on what is selected. Examples:" +
         "<table>" +
-        "<row><col><img src=icons/terrain/terrain_group_all_30.png align=left /></col><col>" + _ "Show all kinds of terrain</col></row>" +
-        "<row><col><img src=icons/terrain/terrain_group_water_deep_30.png align=left /></col><col>" + _ "Show only water terrains</col></row>" +
-        "<row><col><img src=icons/terrain/terrain_group_village_30.png align=left /></col><col>" + _ "Show only villages</col></row>" +
+        "<row><col><img src=icons/terrain/terrain_group_all_30.png align=left /></col><col>" + _ "Show all kinds of terrain" + "</col></row>"
+        "<row><col><img src=icons/terrain/terrain_group_water_deep_30.png align=left /></col><col>" + _ "Show only water terrains" + "</col></row>"
+        "<row><col><img src=icons/terrain/terrain_group_village_30.png align=left /></col><col>" + _ "Show only villages" + "</col></row>"
```

Comment:
Uhh…? Something seems off here. Why don't the lines end in a `+` anymore…? And they still end in a quote? Isn't that a syntax error?

Reply:
Never mind, looks like you fixed it already.